scripts/bench/package.json

```diff
@@ -7,15 +7,15 @@
   },
   "dependencies": {
     "chalk": "^2.1.0",
-    "chrome-launcher": "^0.10.5",
+    "chrome-launcher": "^0.13.2",
     "cli-table": "^0.3.1",
-    "http-server": "^0.10.0",
+    "http-server": "^0.13.0",
     "http2": "^3.3.6",
-    "lighthouse": "^3.2.1",
+    "lighthouse": "^8.1.0",
```

Comment:
This is a massive jump in versions. Have you verified that it still works, or is it a guess? If we're making changes, we should be testing what we've changed. This PR leaves the "test plan" empty which gives the impression that the changes were not tested.